wordpress-mobile/WordPress-iOS
# Implement new Link UI with support for Target and link to other posts.

Fixes #7670 

This PR refactors the UI/UX for the insert links on the Aztec editor.

Here are some screenshots of the functionality:

iPhone:

<img width="462" alt="screen shot 2018-08-28 at 16 56 45" src="https://user-images.githubusercontent.com/651601/44735122-77dbed00-aae3-11e8-8388-870f5a787ab7.png">

iPad:

<img width="1008" alt="screen shot 2018-08-28 at 16 59 47" src="https://user-images.githubusercontent.com/651601/44735251-ce492b80-aae3-11e8-8423-3f008070c6c6.png">


To test:
 - Open and/or create a new post
 - Add a new link to the post
 - Remove a link to the post
 - Update a link on the post
 - Check if the resulting HTML is correct in all scenarios.
